### DISCUSSION TITLE
For some reason I can't login on osu!lazer and I don't know how to fix it

### DISCUSSION THREAD

**Discussion by Garnacho07** [ppy/osu#25390] [Q&A]
When I want to login in osu!lazer a message of error appears, I already see a post with the same problem but I can't find a solution to the problem in that discussion 😕
Also adjunct proff 
![IMG_20231107_223648](https://github.com/ppy/osu/assets/150215216/f4f50d5f-a65a-48c0-a7a7-30f48a75c1ff)
And the link of the other discussion
https://github.com/ppy/osu/issues/3981#issue-395101116

**Comment by peppy** [comment]
Please [provide logs](https://github.com/ppy/osu/discussions/15188) to help us understand your issue.
